### PR TITLE
Update modals, harvester, loader

### PR DESCRIPTION
**Pull request by ivanmrsulja**
## Description

Added necessary update modals for every section of every entity that can be accessed from entity's landing page. Refined notification system. Added harvester. Added loader for journal publications. Fixed A LOT of technical debts.

## How should this be tested?

Only manual tests are to be done.

## Checklist

-   [x] I have tested these changes locally
-   [x] I have added tests that cover these changes (if applicable)
-   [x] I have updated the documentation (if applicable)
-   [x] I have added or updated the necessary comments in the code

### CODE REVIEW

File: src/components/event/update/EventUpdateForm.vue [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                    </v-col>
+                </v-row>
+                <v-row>
+                    <v-checkbox v-model="serialEvent" :label="$t('serialEventLabel')"></v-checkbox>
+                </v-row>
+            </v-col>
+        </v-row>
+
+        <v-row>
+            <p style="margin-left: 20px;">
```
Comment (chenejac) [inline src/components/event/update/EventUpdateForm.vue:75]: might be problematic later for improvements of design, better to move to css file

File: src/components/event/update/EventUpdateModal.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+.edit-pen-container:hover .edit-pen {
+    opacity: 0.3;
+}
+
+.edit-pen-container .edit-pen:hover {
+    opacity: 1;
+}
+</style>
```
Comment (chenejac) [inline src/components/event/update/EventUpdateModal.vue:90]: to move to a css file?

File: src/components/organisationUnit/update/OrganisationUnitRelationUpdateModal.vue [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.edit-pen-container:hover .edit-pen {
+    opacity: 0.3;
+}
+
+.edit-pen-container .edit-pen:hover {
+    opacity: 1;
+}
+</style>
```
Comment (chenejac) [inline src/components/organisationUnit/update/OrganisationUnitRelationUpdateModal.vue:94]: css file

File: src/components/organisationUnit/update/OrganisationUnitUpdateForm.vue [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                        <open-layers-map ref="mapRef" :read-only="false" :init-coordinates="[presetOU?.location?.longitude as number, presetOU?.location?.latitude as number]"></open-layers-map>
+                    </v-col>
+                </v-row>
+            </v-col>
+        </v-row>
+
+        <v-row>
+            <p style="margin-left: 20px;">
+                {{ $t("requiredFieldsMessage") }}
+            </p>
```
Comment (chenejac) [inline src/components/organisationUnit/update/OrganisationUnitUpdateForm.vue:38]: css file

**Review by chenejac — CHANGES_REQUESTED** [review]
@ivanmrsulja please check my comments

File: src/components/event/update/EventUpdateForm.vue (hunk 1 above)
Comment (ivanmrsulja) [reply src/components/event/update/EventUpdateForm.vue:75]: You are right, I will refactor the codebase and replace these occurences of style for required fields message

**Review by chenejac — APPROVED** [review]

**Review by chenejac — APPROVED** [review]

**Pull request merged**